### DISCUSSION TITLE
Can't apply for Student Pack

### DISCUSSION THREAD

**Discussion by TungusSs** [community/community#54168] [GitHub Education] [answered]
Hello, I applied on April 26, 2022 for the Github Student Developer Pack. Since then, exactly one year has passed, and my application has expired, and I need to be revalidated in order to continue to enjoy all the benefits for my studies as a programmer. However, after visiting "Reverify your academic affiliation" page I still see the following message: "Hi, TungusSs! You were last verified as a student on Apr 26, 2022. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits."
Because of this, I cannot reapply. Can you help me with this?

**Comment by ahhzaky** [comment]
If your profile still has Highlights PRO, it looks like you don't need to re-apply because your github student pack is already active.

**Reply by TungusSs** [reply]
> If your profile still has Highlights PRO, it looks like you don't need to re-apply because your github student pack is already active.

Well, yes, but my JetBrains student license, for example, has already expired, and when I try to log in through my GitHub account to renew that license, nothing happens. So I have to manually update the license already through the JetBrains system

**Reply by ahhzaky** [reply]
If your jetbrains license account you have to re-apply at https://www.jetbrains.com/community/education/#students whereas if your github account expires you can re-apply as usual on the github student pack.

**Reply by TungusSs** [reply]
> If your jetbrains license account you have to re-apply at https://www.jetbrains.com/community/education/#students whereas if your github account expires you can re-apply as usual on the github student pack.

That's right. However, after I authorize through GitHub, I am asked for some data, such as mail, last name and first name, etc.
When I specify my email, I get the following error: "This GitHub account has already been used for a similar application. Check your email: you may have already received instructions from us on what to do next."
However, I didn't get any instructions, just messages notifying me about how long before my license expires, that's all.

**Comment by TungusSs** [comment, marked as answer]
Greetings. I have solved my problem. I wrote directly to JetBrains and GitHub support. JetBrains recommended to send them the application not through the Student Pack, but by attaching a physical document, such as a student ID card, and wait about 5 days for them to manually check it.
This worked, I already got access to their products, GitHub support in turn has not responded yet, although it's been about 5 days since the last response. So if anything, know that it's worth it to just apply another way rather than trying to do it through GitHub.

**Reply by ahhzaky** [reply]
Nice👍👍

**Comment by Himanshu2041018132** [comment]
Hey I have been applying for student developer pack from 1 month but they aren;t accepting any of my legal  document. What will be the solution for this though it hit hard that I'm not a privileged student to use  benifits of this program.Please help me with this matter.

**Comment by TungusSs** [comment]
> Hey I have been applying for student developer pack from 1 month but they aren;t accepting any of my legal  document. What will be the solution for this though it hit hard that I'm not a privileged student to use  benifits of this program.Please help me with this matter.

Hi! I had the same problem. To solve it, I just wrote to support that my documents were not accepted by the system, and then they answered me that next time they would manually check my application, thanks to this I received the Student Pack
